src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -1452,4 +1466,19 @@ public Utterance publishUtterance(Utterance utterance) {
     return utterance;
   }
 
+  @Override
+  public String translate(String text) {
+    Response response = getResponse(text);

```

Comment:
This should call the translate method with the max complexity args.

Reply:
removed